javafmllanguage/src/main/java/net/minecraftforge/fml/javafmlmod/AutomaticEventSubscriber.java

```diff
@@ -41,6 +43,11 @@ public static void inject(ModContainer mod, ModFileScanData scanData, ClassLoade
             .filter(data -> AUTO_SUBSCRIBER.equals(data.annotationType()))
             .toList();
 
+        var onlyIns = scanData.getAnnotations().stream()
```

Comment:
Please put this inside a userdev check to avoid unnecessary processing in prod. Can use `Collections.emptySet()` when in prod.

Reply:
Done